shared_model/bindings/model_crypto.cpp

```diff
@@ -28,6 +28,9 @@ namespace shared_model {
 
     crypto::Keypair ModelCrypto::fromPrivateKey(
         const std::string &private_key) {
+      if (private_key.length() != 32)  {
```

Comment:
My bad: size must be equal to 64 obviously (32 bytes means 64-length string when hex-encoded). Please change here and in other places.